README.md

```diff
@@ -34,24 +35,25 @@ Lets make a concrete example:
  PO moves this to TODO.
 
  A developer PULLS this task, means: moves this task from TODO into IN PROGRESS>
- Once she is done, set the task into "COMPLETED the current step". So small detail:
+ Once He/She is done, set the task into "COMPLETED the current step". So small detail:
```

Comment:
She/he at least please :)

Reply:
Made the changes.

Reply:
I can squash and merge this, but we need to fix the check that's failing. Let me fix that and merge.